src/main/java/org/example/bookstore/controller/BookControllerImpl.java

```diff
@@ -0,0 +1,38 @@
+package org.example.bookstore.controller;
+
+import java.util.List;
+import lombok.AllArgsConstructor;
+import org.example.bookstore.dto.BookDto;
+import org.example.bookstore.dto.CreateBookRequestDto;
+import org.example.bookstore.service.BookService;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Component
+@RestController
+@AllArgsConstructor
+@RequestMapping(value = "/books")
```

Comment:
```suggestion
@RequestMapping(value = "/api/books")
```
GET: /api/books (Retrieve book catalog) In task description